tracer/test/Datadog.Trace.Tests/RuntimeMetrics/RuntimeMetricsWriterTests.cs

```diff
@@ -157,8 +157,8 @@ public async Task ShouldCaptureProcessMetrics()
 
                 actualNumberOfThreads.Should().NotBeNull();
 
-                // To future generations: if 100 is not enough, feel free to bump it up. We're really just checking that the value is "realistic".
-                actualNumberOfThreads.Should().NotBeNull().And.BeGreaterThan(0).And.BeInRange(expectedNumberOfThreads - 100, expectedNumberOfThreads + 100);
+                // A margin of 500 threads seem like a lot, but we have tests that spawn a large number of threads to try to find race conditions
+                actualNumberOfThreads.Should().NotBeNull().And.BeGreaterThan(0).And.BeInRange(expectedNumberOfThreads - 500, expectedNumberOfThreads + 500);
```

Comment:
Is this going to be safer? It feels like we're mostly just checking "is a number" here 😅 
```suggestion
                actualNumberOfThreads.Should().NotBeNull().And.BeGreaterThanOrEqualTo(expectedNumberOfThreads );
```

Reply:
"Is a number bigger than 0 and not too crazy" 😛